### PR TITLE
Filesystem Notifications - UI

### PR DESCRIPTION
**Pull request by imsodin**
This is about the changes in the web UI due to #3986.

Not that this is based on the branch plouj-fswatcher and not master to visually separate the changes. It is generally not polished yet - thus should be considered WIP.

### CODE REVIEW

File: gui/default/assets/lang/lang-en.json [hunk 1]
```diff
@@ -268,6 +272,7 @@
    "Warning, this path is a parent directory of an existing folder \"{%otherFolderLabel%}\" ({%otherFolder%}).": "Warning, this path is a parent directory of an existing folder \"{{otherFolderLabel}}\" ({{otherFolder}}).",
    "Warning, this path is a subdirectory of an existing folder \"{%otherFolder%}\".": "Warning, this path is a subdirectory of an existing folder \"{{otherFolder}}\".",
    "Warning, this path is a subdirectory of an existing folder \"{%otherFolderLabel%}\" ({%otherFolder%}).": "Warning, this path is a subdirectory of an existing folder \"{{otherFolderLabel}}\" ({{otherFolder}}).",
+   "Warning: If you are using an external watcher like {%syncthingInotify%}, you should to deactivate it.": "Warning: If you are using an external watcher like {{syncthingInotify}}, you should to deactivate it.",
```
Comment (TwP) [inline gui/default/assets/lang/lang-en.json]: :eyes: grammar: `, you should to deactivate it.` Is the "to" there intentional?

File: gui/default/syncthing/core/notifications.html [hunk 2]
```diff
@@ -40,4 +40,29 @@ <h3 class="panel-title"><span class="fa fa-flash"></span>&nbsp;<span translate>A
...
+<notification id="fsNotifyNotification">
+  <div class="panel panel-success">
+    <div class="panel-heading">
+      <h3 class="panel-title"><span class="fa fa-flash"></span>&nbsp;<span translate>Filesystem Notifications</span></h3>
+    </div>
+    <div class="panel-body">
+      <p translate translate-value-fs-notifications="{{'Filesystem notifications' | translate}}">{%fsNotifications%} are now available. This will detect changes on disk and issue a scan. The benefits are that changes are propagated quicker and that less full scans are required.</p>
+      <p><a href="https://docs.syncthing.net/users/TBD"><span class="fa fa-fw fa-book"></span>&nbsp;<span translate>Learn more</span></a></p>
+      <p translate translate-value-fs-notifications="{{'filesystem notifications' | translate}}">Do you want to activate {%fsNotifications%} for all your folders?</p>
+      <p translate translate-value-syncthing-inotify="syncthing-inotify">Warning: If you are using an external watcher like {%syncthingInotify%}, you should to deactivate it.</p>
```
Comment (TwP) [inline gui/default/syncthing/core/notifications.html]: :eyes: grammar: same extraneous "to" here as well.

**Comment by lkwg82** [comment]
the referenced branch already merged

**Comment by AudriusButkevicius** [comment]
UI branch hasn't been merged yet.

**Comment by lkwg82** [comment]
I meant the 'plouj-fswatcher', not the UI branch.

**Comment by calmh** [comment]
Is it time to revive this and close off the notification stuff?

**Pull request closed without merging**